scss/_patterns_side-navigation.scss

```diff
@@ -202,7 +202,7 @@
 
     // vf-highlight-bar is rendered above focus outline
     // so we need to hide it on focus
-    &:focus::before {
+    &:focus-visible::before {
```

Comment:
`:focus-visible` is only supported by very latest browsers, we need to keep fallback `:focus` for the others.

Similar as we do in the mixin:
https://github.com/canonical-web-and-design/vanilla-framework/blob/57cdc27c42c892c9d834826fcd62f0e3682bfad2/scss/_global_functions.scss#L64-L77

(so you need to add focus styles both for `focus` and `focus-visible` states, but then remove them from `:focus:not(:focus-visible)`, as in the snippet above.

Reply:
Thanks, I've reversed these to also handle `:focus`.